notebooks/hive/PyHive.ipynb

```diff
@@ -0,0 +1,726 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Exploration and Reporting with PyHive"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example we show how to explore data in Hive and build reports.\n",
+    "\n",
+    "The example uses a dataset of real estate sales in the Sacramento area, which you can download from [here](http://snurran.sics.se/hops/hive/Sacramentorealestatetransactions.csv).\n",
+    "Create a dataset in your project (without README.md) and upload the CSV file in there.\n",
+    "\n",
+    "First thing we need to setup the IPython magic and the connection information by running the following cell:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%reload_ext sql\n",
+    "from hops import hive\n",
+    "hive.setup_hive_connection()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load Data into an External table"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's first create an Hive external and let's point the table to the just created dataset. In the query below you should replace `[Projectname]` with your project name.\n",
+    "\n",
+    "The `%%sql` magic allows you to write HiveQL and execute the query against the HiveServer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%sql\n",
+    "CREATE EXTERNAL TABLE sacramento_properties_ext(\n",
+    "street string,\n",
+    "city string,\n",
+    "zip int,\n",
+    "state string,\n",
+    "beds int,\n",
+    "baths int,\n",
+    "sq__ft float,\n",
+    "sales_type string,\n",
+    "sale_date string,\n",
+    "price float,\n",
+    "latitude float,\n",
+    "longitude float)\n",
+    "ROW FORMAT DELIMITED\n",
+    "FIELDS TERMINATED BY ','\n",
+    "LOCATION '/Projects/[Projectname]/RawData'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To make sure the creation succeded we can list all the tables in the project's database with and run a simple query agains the external table"
```

Comment:
@SirOibaf `To make sure the table was created successfully, we can list all the tables in the project's database and run a simple query against the external table.